ui/main.qml

```diff
@@ -53,6 +53,7 @@ StatusWindow {
         property bool isBrowserEnabled: false
         property bool isActivityCenterEnabled: false
         property bool showOnlineUsers: false
+        property bool expandUsersList: false
```

Comment:
I wonder why this was gone in the first place? Because this used to work... has something changed in the settings structure lately?

I've also noticed that some of my settings were gone after latest update..